glide.yaml

```diff
@@ -2,12 +2,12 @@ package: github.com/uber/jaeger-lib
 import:
 - package: github.com/codahale/hdrhistogram
 - package: github.com/go-kit/kit
-  version: v0.5.0
+  version: v0.9.0
   subpackages:
   - metrics/influx
 - package: github.com/uber-go/tally
   version: '>= 2.1.0, < 4'
 - package: github.com/prometheus/client_golang
-  version: v0.8.0
+  version: v1.1.0
```

Comment:
I think for glide, you still want a version range rather than an exact version.
`^1` should suffice for client_golang and `~0.9.0` for go-kit.

(If I remember correctly, "X.Y.Z" does the right thing in dep so `~` or `^` aren't needed in the Gopkg.toml.)

Reply:
+1